### DISCUSSION TITLE
[Q] Systemless cert for trust user cert

### DISCUSSION THREAD

**Discussion by JeelsBoobz** [tiann/KernelSU#1353] [Q&A]
Is it no longer possible for latest ksu to make a user cert trusted, such as creating a /system/etc/security/cacerts module?
I have tried but it still reads untrusted

**Comment by devnoname120** [comment]
Android 14 uses `conscrypt` and ignores the `cacerts` from that path.

See here for a solution: https://github.com/AdguardTeam/adguardcert/blob/9b0fe1e0907228a2dd69e4b0fe9cac848add336a/module/post-fs-data.sh#L46-L73
